resolwe_bio/processes/workflows/bbduk_star_htseq.yml

```diff
@@ -1,33 +1,40 @@
 - slug: workflow-bbduk-star-htseq
```

Comment:
We should append `-single` to the slug of this workflow for a consistent naming scheme.